prime-angular-frontend/src/app/core/services/utils.service.ts

```diff
@@ -85,6 +85,39 @@ export class UtilsService {
         ? -1 : 0;
   }
 
+  /**
+   * @description
+   * Generic sorting of a JSON object by key with asc flag
+   */
+  public sortBy<T>(a: T, b: T, isAsc: boolean): number {
+    return this.sort(a, b) * (isAsc ? 1 : -1);
+  }
+
+  /**
+   * @description
+   * Generic sorting of a JSON object by key with nulls at end
+   */
+  public sortWithNullsAtEnd<T>(a: T, b: T): SortWeight {
```

Comment:
This might be better as `sortWithTrailingNulls(...)` if we want to have a specific method for sorting `null`s, but feels like we could just by default always sort `null` the same way in `sortByOrder(...)` and might not need this method